resilience4j-spring-boot-common/src/main/java/io/github/resilience4j/retry/autoconfigure/AbstractRetryConfigurationOnMissingBean.java

```diff
@@ -91,7 +91,7 @@ public RetryRegistry retryRegistry(RetryConfigurationProperties retryConfigurati
     @ConditionalOnMissingBean
     public RetryAspect retryAspect(RetryConfigurationProperties retryConfigurationProperties,
         RetryRegistry retryRegistry,
-        @Autowired(required = false) List<RetryAspectExt> retryAspectExtList,
+        @Autowired(required = false) List<RetryDecoratorExt> retryAspectExtList,
```

Comment:
Could you change variable name? 
```suggestion
        @Autowired(required = false) List<RetryDecoratorExt> retryDecoratorExtList,
```